changelog/62754.fixed

```diff
@@ -0,0 +1 @@
+Deprecate core ESXi and associated states and modules, vcenter and vsphere support in favor of Salt VMware Extensions
```

Comment:
The file extension should be `.deprecated` since you are deprecating modules.

Reply:
Thanks for the catch, fixed it